virt/virt-4-8-release-notes.adoc

```diff
@@ -214,3 +214,35 @@ No model registered for VirtualMachines
 ----
 +
 You can ignore this message. It does not affect VM import. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974812[*BZ#1974812*])
+
+* When creating a virtual machine after upgrading to OCP 4.8 while running CNV 2.6.5, various errors occur.
++
+** Selecting *Create* -> *With YAML* on the *Virtualization* page of the {VirtProductName} console displays the following warning message:
++
+[source]
+----
+The server doesn't have a resource type "kind: VirtualMachine, apiVersion: kubevirt.io/v1"
+----
++
+*** As a workaround, modify the `apiVersion` in the manifest. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979114[*BZ#1979114*])
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3 <1>
+kind: VirtualMachine
+metadata:
+  annotations:
+...
+----
+<1> The modified `apiVersion: kubevirt.io/v1alpha3`
++
+** Creating a virtual machine from the Customize wizard provides a unclear and non-specific error message:
++
+[source]
+----
+Error creating virtual machine
+----
++
+*** As a workaround, copy the manifest and create the virtual machine from the CLI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
++
+** When connecting to the VNC console by using the {VirtProductName} web console and `virtctl` command, sometimes the VNC console fails to respond. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
```

Comment:
In the UI it's not sometimes, AFAIK the CLI doesn't have this problem
Using UI OCP-version-4.8.0 you can not connect at all to a VM running on CNV-version-2.6
workaround - use CLI, or upgrade to CNV-version-4.8.0

Reply:
note: a fixed is planed for OCP-4.8.1, and then we could VNC into a VM running on CNV-2.6.z

Reply:
I've made changes based on the comments. What do you think?